lib/kitchen/driver/ec2.rb

```diff
@@ -341,6 +342,15 @@ def block_device_mappings
 
         bdms
       end
+
+      # TODO Update this to use the new Transport logic when that is ready
+      def create_ec2_json(state)
+        Kitchen::SSH.new(*build_ssh_args(state)) do |conn|
+          run_remote("sudo mkdir -p /etc/chef/ohai/hints;sudo touch /etc/chef/ohai/hints/ec2.json",
```

Comment:
This is not an ideal fix - when we can leverage the Transport logic from the windows-guest-support branches we will move this logic there.  This is just a temporary workaround until we get access to that.

Having said that, do you think I should add a feature flag or something, so people on systems without sudo (or `touch`) aren't totally borked by this?